content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -393,7 +393,8 @@ The feature life cycle matched to its corresponding feature gate is:
   * GA: the feature gate is deprecated (see ["Deprecation"](#deprecation)) and becomes
   non-operational.
   * GA, deprecation window complete: the feature gate is removed and calls to it are
-  no longer accepted.
+  no longer accepted. Feature gate is disabled by default and can be enabled by
+  the user during the deprecation window.
```

Comment:
I think this is wrong. One a feature is fully stable, you cannot set the feature gate  that previously controlled it. In fact, attempting to do so may be treated as an error.

In this phase the feature is moving from _stable and the feature gate setting is ignored_ to _stable and the feature gate setting is invalid_.

This text is not about deprecating stable features. If we did add a feature (let's say `WatchBookmark`), and then deprecated it, the `WatchBookmark` feature gate would stay removed. We would leave the watch bookmark behavior in the API but we'd deprecated it, separately, and might warn people when they specify query strings etc that involve it.

Possibly but unlikely: we might add a new feature `ForbidWatchBookmark` that when enabled always returns a 403 should someone try to use a watch bookmark. This kind of deprecation-and-feature-removal is so rare that I don't have an example.

Reply:
hm, GA'd feature gate cannot be deprecated as per the next section. I think this section calls removal of a feature gate and locking it to `true` with the term "deprecation". Where this deprecation is only deprecates the feature gate use. Which is very confusing as can be mistaken with the feature deprecation. Let me rephrase